src/class/cdc/cdc_device.c

```diff
@@ -87,6 +87,24 @@ typedef struct
 //--------------------------------------------------------------------+
 CFG_TUSB_MEM_SECTION static cdcd_interface_t _cdcd_itf[CFG_TUD_CDC] = { { 0 } };
 
+bool pending_read_from_host;
+static void _prep_next_transaction(void) {
+    uint8_t const itf = 0;
+    cdcd_interface_t* p_cdc = &_cdcd_itf[itf];
+
+    // skip if previous transfer not complete
+    if (pending_read_from_host) {

```

Comment:
we can use dcd_edpt_busy() function for checking if we already queued an IN transfer